README.md

```diff
@@ -101,13 +109,15 @@ files.
 ## See also
 
 - https://www.w3.org/2013/csvw/wiki/Main_Page
+- https:csvw.org
```

Comment:
`https://csvw.org` (did not know there was a website on csvw, cool)

Reply:
Yes, I was made aware of this just recently by the csvwr maintainer https://github.com/Robsteranium/csvwr/issues/5#issuecomment-1154834195

Reply:
You should just add the two `//` to the URL ;)